api/src/integration/java/org/pmiops/workbench/BaseIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package org.pmiops.workbench;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Scope;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+@RunWith(SpringRunner.class)
+@WebAppConfiguration
+@Import({IntegrationTestConfig.class})
+public abstract class BaseIntegrationTest {
+
+  static WorkbenchConfig config;
```

Comment:
package private doesn't seem correct here - is the intent that subclasses can modify this? If so, probably should be public or protected (likewise for `loadConfig`)